File: beecrowd/Java/1010.java
import java.io.IOException;
import java.util.Scanner;

public class Main{
    public static void main(String [] args) throws IOException{
        Scanner keyboard = new Scanner (System.in);

        //lê valores da primeira linha
        int codigo1 = keyboard.nextInt();
        int qt1 = keyboard.nextInt();
        double preco1 = keyboard.nextDouble();

        //lê a segunda linha
        int codigo2 = keyboard.nextInt();
        int qt2 = keyboard.nextInt();
        double preco2 = keyboard.nextDouble();

        //calcula o total
        double total = (qt1 * preco1) + (qt2 * preco2);

        //printa na tela o total
        System.out.printf("VALOR A PAGAR: R$ %.2f\n", total);

        keyboard.close();
    }
}

File: beecrowd/C++/1010.cpp
#include <iostream>
#include <iomanip>
using namespace std;

int main()
{

    int cod1, cod2, qt1, qt2;
    double preco1, preco2;

    cin >> cod1 >> qt1 >> preco1;
    cin >> cod2 >> qt2 >> preco2;

    double total = (qt1 * preco1) + (qt2 * preco2);

    cout << fixed << setprecision(2);
    cout << "VALOR A PAGAR: R$ " << total << endl;

    return 0;
} // end main